lib/rexml/source.rb

```diff
@@ -321,7 +321,7 @@ def current_line
         rescue
         end
         @er_source.seek(pos)
-      rescue IOError
+      rescue IOError, Errno::EPIPE, Errno::ESPIPE, Errno::EINVAL
```

Comment:
We can use `SystemCallError` that is an ancestor class of them:

```suggestion
      rescue IOError, SystemCallError
```

Reply:
OK.
I see.